themes/default/content/challenge/startup-in-a-box.md

```diff
@@ -39,14 +71,37 @@ Follow along with the steps outlined on this page.
 
 #### Step 1. Your First Pulumi Program
 
-You will learn how to create a new Pulumi program using our Pulumi templates, specifically for AWS with TypeScript. Create a new directory called `pulumi-challenge` and run the following inside of it:
+You will learn how to create a new Pulumi program using our Pulumi templates, specifically for AWS or Google Cloud with TypeScript. Create a new directory called `pulumi-challenge` and run the following inside of it:
+
+{{% chooser cloud "aws,gcp" %}}
+
+{{% choosable cloud aws %}}
 
 ```shell
 pulumi new aws-typescript
 ```
 
+{{% /choosable %}}
+
+{{% choosable cloud gcp %}}
+
+```shell
+# create a new Pulumi project
+pulumi new gcp-typescript
+
+# set the project ID
+pulumi config set gcp:project <GCP_PROJECT_ID>
```

Comment:
I've reworded this section to say that the new project wizard will prompt for the necessary information, but I show the `pulumi config set` commands in case the user needs to change the values or didn't enter a value.